packages/components/package.json

```diff
@@ -36,8 +36,14 @@
   },
   "dependencies": {
     "@kaizen/component-library": "^16.5.1",
+    "@kaizen/design-tokens": "^10.3.1",
```

Comment:
It's already a peer dep, is this necessary?

Reply:
Nope! I don't know where this came from :D